mitmproxy/io/compat.py

```diff
@@ -30,6 +30,7 @@ def convert_013_014(data):
         str(x) for x in data[b"response"].pop(b"httpversion")).encode()
     data[b"response"][b"status_code"] = data[b"response"].pop(b"code")
     data[b"response"][b"body"] = data[b"response"].pop(b"content")
+    data[b"response"][b"trailers"] = None
```

Comment:
Please move this to a new converter function - otherwise the whole versioning of flow files gets messed up.

At the bottom of this file is a list / mapping of converter functions and version numbers. 
You can add a new one (7 to 8) and then bump the `version.FLOW_FORMAT_VERSION` accordingly.